src/components/listitems/ListItemInfoCopy.tsx

```diff
@@ -30,9 +30,11 @@ export type ListItemInfoCopy = WithTestID<{
   numberOfLines?: number;
   onPress: (event: GestureResponderEvent) => void;
   icon?: IOIcons;
-  // Accessibility
-  accessibilityLabel: string;
-}>;
+}> &
+  Pick<
+    ComponentProps<typeof Pressable>,
+    "accessibilityLabel" | "accessibilityHint"
+  >;
```

Comment:
I would leave the label here as mandatory, since we should highlight that this button aims to copy the value and the I18n value of the "label" would not be available here but in the Main App I18n config. What do you think?

Reply:
ok after further analysis we may leave the copy instruction as a hint, specified by the main app